mappings/net/minecraft/resource/pack/ResourcePack.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/unmapped/C_tguinuvn net/minecraft/resource/pack/ResourcePack
 	COMMENT A resource pack, providing resources to resource managers.
 	COMMENT <p>
-	COMMENT They are single-use in each reload cycle of a reloadable resource manager.
+	COMMENT ResourcePacks are single-use in each reload cycle of a reloadable resource manager.
```

Comment:
Hmm, shouldn't this be `{@code }`'d? I don't see much wrong with this

Reply:
I'd go even farther and `{@link ResourcePack ResourcePacks}` it

Reply:
lol, I'm not linking it to itself, but I'll add the code decoration

Reply:
```suggestion
	COMMENT {@code ResourcePack}s are single-use in each reload cycle of a reloadable resource manager.
```